benchmarks/operator_benchmark/pt/qobserver_test.py

```diff
@@ -84,7 +113,9 @@ def init(self, C, M, N, dtype, qscheme, op_func, device):
         self.op_func = op_func(dtype=dtype, qscheme=qscheme).to(device)
 
     def forward(self):
-        return self.op_func(self.f_input)
+        self.op_func(self.f_input)
```

Comment:
Might be good to benchmark the forward and the calcQparams separately. forward gets called every iteration, while qparams gets called only once. Having separate metrics will help figure out the bottlenecks better

Reply:
this is making the benchmark represent what happens inside the observer during QAT, not keeping the old code around because I'm not aware of a need for it in the near future.  We have separate benchmarks for histogram observers, and I'm not aware of any requests to optimize observers outside of QAT + histogram observers.

Reply:
calculate_qparams is called at every pass through the observer during QAT, when observers are enabled